### ISSUE TITLE
height & min-row-height

### ISSUE THREAD

**Issue by Ancadea** [ericferreira1992/ag-virtual-scroll#14]
Hey,

Thank you for your ag-virtual-scroll package!
I have two questions: 
1.) The min-row-height is not working. I always receive this error message:  Type 'string' is not assignable to type 'number'.
2.) Is it possible to use percentage values in height? 

Thank you

**Comment by ericferreira1992** [comment]
Hello @Ancadea, thank you!

Answers:
1.) I cant simulate that error here, can you send me an example and Angular version you're using?
2.) Do you talking about prop `height`? If yes, you can. But if you talking about `min-row-height`, you kinda can, but it won't work, because the that prop is used to indicate a minumun height of each row, for the virtual-scroll works well, then percentages values doesn't seem a good thing.

**Comment by Ancadea** [comment]
Hey Eric,

Thank you for your response. 

1.) I am using angular 14. If I use min-row-height the application doesn't compile. 
<ag-virtual-scroll #vs [items]="caseItems" height="100%" min-row-height="50" class="box-border">
  <ui-case-list-item
    *ngFor="let item of vs.items"
    [case]="item"
    >
  </ui-case-list-item>
</ag-virtual-scroll>
If I use 100% in height, I only can see 3 items. I think it because of the min-row-height. Did you have a similar problem?

Thank you and best regards

**Comment by ericferreira1992** [comment]
Maybe It could be happening because of error from `min-row-height`. But, yesterday I released a new version of ag-virtual-scroll (v1.5.0) wich is updated with v14 of Angular. Try to update for a new version, and see if the error still ocours.

I wait for you.

**Comment by Ancadea** [comment]
Thank you! That helped.

**Comment by ericferreira1992** [comment]
Ancadea, it all right? If you don't post here again, I'll close this issue soon.

**Comment by dePablobernat** [comment]
Hello @ericferreira1992 I'm using your virtual-scroll package and works as a charm. But still I have the same issue as @Ancadea commented. If I set the element height to 100% I'm only able to see 3 items. If i scroll a tiny bit it autosizes the column to the expected height. 
What could be happening?

My list is displayed on a container with auto-calculated height and if I set the element .item-container to be 100% in css does fill the space but only 3 elements are rendered.

```
<ag-virtual-scroll
  #vs
  [items]="items"
  min-row-height="109"
  height="100%"
  class="box-border"
>
   <item-card
      *ngFor="let item of vs.items; trackBy: trackByItemFn"
      [attr.data-testid]="'item-card'"
      [card]="item"
      (itemClicked)="
         onClickItemCard({ event: $event, columnId: groupItem.id })
       "
       [hiddenDeleteButton]="
          facade.filters.grouped_by !== 'aggregation_level'
       "
       (deleteItemClicked)="onClickDeleteItemCard($event, groupItem.id)"
    >
    </item-card>
 </ag-virtual-scroll>
```

I've been testing with the released version 1.5.2 with angular 14+
Thank you very much!
